app/helpers/expenses_helper.rb

```diff
@@ -0,0 +1,14 @@
+module ExpensesHelper
+  def format_date(date)
+    date.strftime("%d %B, %Y")
+  end
+
+  def display_ajax_success(message)
+    html = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.